packages/vue-query/src/__mocks__/useQueryClient.ts

```diff
@@ -1,9 +1,10 @@
 import { QueryClient } from '../queryClient'
+import { vi } from 'vitest'
 
 const queryClient = new QueryClient({
   defaultOptions: {
     queries: { retry: false, gcTime: Infinity },
   },
 })
 
-export const useQueryClient = jest.fn(() => queryClient)
+export const useQueryClient = vi.fn(() => queryClient)
```

Comment:
This will be used by `automocking` - https://vitest.dev/guide/mocking.html#automocking-algorithm